internal/pkg/version/version.go

```diff
@@ -4,5 +4,25 @@
 // Package version holds variables for generating version information
 package version
 
+const (
+	// LegacyAppTemplateVersion is the version associated with the application template before we started versioning.
+	LegacyAppTemplateVersion = "v0.0.0"
+	// AliasLeastAppTemplateVersion is the least version number available for HTTPS alias.
+	AliasLeastAppTemplateVersion = "v1.0.0"
+	// StaticSiteMinAppTemplateVersion is the minimum app version required to deploy a static site.
+	StaticSiteMinAppTemplateVersion = "v1.2.0"
```

Comment:
nit: to have constant names... maybe. Just trying to imagine what would I type when I need to use this variable while coding 💭 I might just type "version.App" or "version.MinV" and hope the IDE can give me the correct suggestions

```suggestion
         AppTemplateMinVersionAlias = "v1.0.0"
	// StaticSiteMinAppTemplateVersion is the minimum app version required to deploy a static site.
	AppTemplateMinVersionStaticSite = "v1.2.0"
```